code/datums/components/storage/storage.dm

```diff
@@ -374,6 +375,8 @@
 	closer.screen_loc = "[screen_start_x + cols]:[screen_pixel_x],[screen_start_y]:[screen_pixel_y]"
 
 /datum/component/storage/proc/show_to(mob/M)
+	if(!can_be_opened)
+		return FALSE
```

Comment:
Can we show a user a message here to say why they cannot open it and that its not a bug?

Reply:
I cannot fathom a good IC message to send here lol

Reply:
I don't really mind if it isn't an IC message, something like "You cannot open this!" as an overhead message works fine

Reply:
Addressed now.